GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -403,15 +431,32 @@ private void tvGitTree_ItemDrag(object sender, ItemDragEventArgs e)
         {
             if (e.Item is TreeNode { Tag: GitItem gitItem })
             {
-                StringCollection fileList = new();
-                var fileName = _fullPathResolver.Resolve(gitItem.FileName);
+                string tempFilePath;
+                try
+                {
+                    tempFilePath = SaveItemToTempFile(gitItem);
+                }
+                catch (UnauthorizedAccessException)
+                {
+                    MessageBox.Show(_cantSaveToTemp.Text, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    return;
+                }
+                catch (FileNotFoundException)
+                {
+                    MessageBox.Show(_cantExtractTempArchive.Text, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    return;
+                }
 
-                fileList.Add(fileName.ToNativePath());
+                if (tempFilePath is not null)
```

Comment:
🤔 `tempFilePath` is not declared as `string?`.
If it really can be `null`, the old variant could be kept as fallback.

Reply:
Sorry, should be `string?`, as it could be null if the GitItem.FileName was empty, but no need for me to check for it as the drag operation is ok to continue with null.